plugins/core/jetbrains-community/src/icons/AwsIcons.kt

```diff
@@ -145,6 +145,30 @@ object AwsIcons {
         @JvmField val CHECKMARK_GRAY = load("/icons/resources/codetransform/checkmark.svg")
 
         @JvmField val TIMELINE_STEP = load("/icons/resources/codetransform/transform-timeline-step-done.svg") // 16 * 16
+
+        @JvmField val PLAN_VARIABLES_LIGHT = load("/icons/resources/codetransform/transform-variables-light.svg")
+
+        @JvmField val PLAN_VARIABLES_DARK = load("/icons/resources/codetransform/transform-variables-dark.svg")
+
+        @JvmField val PLAN_STEP_INTO_LIGHT = load("/icons/resources/codetransform/transform-step-into-light.svg")
+
+        @JvmField val PLAN_STEP_INTO_DARK = load("/icons/resources/codetransform/transform-step-into-dark.svg")
+
+        @JvmField val PLAN_DEPENDENCIES_LIGHT = load("/icons/resources/codetransform/transform-dependencies-light.svg")
+
+        @JvmField val PLAN_DEPENDENCIES_DARK = load("/icons/resources/codetransform/transform-dependencies-dark.svg")
+
+        @JvmField val PLAN_FILE_LIGHT = load("/icons/resources/codetransform/transform-file-light.svg")
+
+        @JvmField val PLAN_FILE_DARK = load("/icons/resources/codetransform/transform-file-dark.svg")
+
+        @JvmField val PLAN_ARROW_LIGHT = load("/icons/resources/codetransform/transform-arrow-light.svg")
+
+        @JvmField val PLAN_ARROW_DARK = load("/icons/resources/codetransform/transform-arrow-dark.svg")
+
+        @JvmField val PLAN_DEFAULT_LIGHT = load("/icons/resources/codetransform/transform-default-light.svg")
+
+        @JvmField val PLAN_DEFAULT_DARK = load("/icons/resources/codetransform/transform-default-dark.svg")
```

Comment:
for reference light and dark can be handled automatically by the icon loader
https://plugins.jetbrains.com/docs/intellij/icons.html#svg-format (see png for scheme)

Reply:
Will follow up with this, good to know.